src/community/backup-restore/core/src/main/java/org/geoserver/backuprestore/writer/ResourceInfoAdditionalResourceWriter.java

```diff
@@ -0,0 +1,93 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.backuprestore.writer;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.geoserver.backuprestore.Backup;
+import org.geoserver.catalog.ResourceInfo;
+import org.geoserver.platform.resource.Files;
+import org.geoserver.platform.resource.Paths;
+import org.geoserver.platform.resource.Resource;
+import org.geoserver.platform.resource.Resources;
+import org.geoserver.template.GeoServerTemplateLoader;
+
+/**
+ * TODO
+ * 
+ * @author Alessio Fabiani, GeoSolutions
+ *
+ */
+public class ResourceInfoAdditionalResourceWriter
```

Comment:
I use exactly the same class for Restore. My fault, the name is confusing. Need to refactor a bit.